tests/Unit/Digitick/Sepa/Util/StringHelperTest.php

```diff
@@ -17,7 +17,7 @@ public function testGermanCharacters()
     {
         $string = 'ÄÖÜäöüß';
 
-        $this->assertEquals('AeOeUeaeoeuess', StringHelper::sanitizeString($string));
+        $this->assertEquals('AEOEUEaeoeuess', StringHelper::sanitizeString($string));
```

Comment:
Same as above the changes are wrong as they indicate change of behaviour of the former implementation